newsroom/default_settings.py

```diff
@@ -255,21 +250,36 @@
 }
 
 
+LANGUAGES = ['en', 'fi', 'cs', 'fr_CA']
+DEFAULT_LANGUAGE = 'en'
+
+CLIENT_LOCALE_FORMATS = {
+    "en": {  # defaults
+        "TIME_FORMAT": "HH:mm",
+        "DATE_FORMAT": "DD/MM/YYYY",
+        "COVERAGE_DATE_TIME_FORMAT": "HH:mm DD/MM",
+        "COVERAGE_DATE_FORMAT": "DD/MM",
+    },
+    "fr_CA": {  # example - you can overwrite any format above
+        "DATE_FORMAT": "DD/MM/YYYY",
+    }
+}
+CLIENT_TIME_FORMAT = 'HH:mm'
+CLIENT_DATE_FORMAT = 'DD/MM/YYYY'
+CLIENT_COVERAGE_DATE_TIME_FORMAT = 'HH:mm DD/MM'
+CLIENT_COVERAGE_DATE_FORMAT = 'DD/MM'
```

Comment:
not sure if we still need these, is it used anywhere?

Reply:
No we don't, I guess I didn't stage that change, I'll fix that 👍